packages/context/README.md

```diff
@@ -28,8 +28,7 @@ app.bind('defaultName').to('John'); // setting context
 
 class HelloController {
   // consider this.ctx here
-  constructor(@inject('defaultName') name) { // injecting dependency using context
-    this.name = name;
+  constructor(@inject('defaultName') private name) { // injecting dependency using context
```

Comment:
I don't think this is valid in either TypeScript or JavaScript. Can we fix this to either have no property access modifiers, or to be valid TypeScript instead? (`@inject('defaultName') private name: string`)